home-work9.py

```diff
@@ -0,0 +1,14 @@
+#create Fibonacci Generator
+def f_generator(number):
+    numb1, numb2 = 0, 1
+    for _ in range(number):
+        yield numb1
+        numb1, numb2 = numb2, numb1+numb2
+
+inp = int(input("Enter integer for Fibonacci Generator "))
+gen1 = f_generator(inp)
+for _ in range(inp):
+    print(next(gen1))
```

Comment:
The task was to print only one number of sequence but not all of them

Reply:
Also, I would suggest to start counting from zero:  0 = 0, 1 = 1, 2 = 1, 3 = 2, 4 = 3, 5 = 5, 6 = 8, 7 = 13

Reply:
https://github.com/LilianaLukash/PythonStudy/pull/6/commits/663793f1bce511b03ebded6cd15d8616ce34a790
Please see file with printing only one number.

Program already was printing from 0